src/main/java/com/demonwav/mcdev/platform/canary/CanaryModuleType.java

```diff
@@ -0,0 +1,68 @@
+package com.demonwav.mcdev.platform.canary;
+
+import com.demonwav.mcdev.platform.AbstractModuleType;
+import com.demonwav.mcdev.platform.PlatformType;
+import com.demonwav.mcdev.platform.canary.util.CanaryConstants;
+import com.demonwav.mcdev.util.CommonColors;
+import com.google.common.collect.ImmutableList;
```

Comment:
Check the style guide, `com.demonwav` imports should be separate. Use [this](https://github.com/DemonWav/MinecraftDev/blob/master/intellij-java-style.xml) to set the style properly in IntelliJ easilly.